.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+jobs:
+  release:
+    if: github.ref == 'refs/heads/master'
+
+    name: Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "10"
```

Comment:
We might want to try updating the Node Version here to 16 too

Reply:
done